lib/api/controllers/securityController.js

```diff
@@ -0,0 +1,79 @@
+var
+  ResponseObject = require('../core/models/responseObject');
+
+module.exports = function SecurityController (kuzzle) {
+
+  /**
+   * Get a specific role according to the given id
+   * @param {RequestObject} requestObject
+   * @returns {Promise}
+   */
+  this.getRole = function (requestObject) {
+    kuzzle.pluginsManager.trigger('data:getRole', requestObject);
+
+    return kuzzle.repositories.role.loadRole(
```

Comment:
Should add a `catch` handler, as `kuzzle.repositories.role.loadRole` may return a rejected promise.